app/serializers/submission_serializer/definitions/offence.rb

```diff
@@ -0,0 +1,13 @@
+module SubmissionSerializer
+  module Definitions
+    class Offence < Definitions::BaseDefinition
+      def to_builder
+        Jbuilder.new do |json|
+          json.name offence_name
```

Comment:
From a `Charge` instance you can get the `Offence` instance with `charge.offence` and then get the name and the class with `charge.offence.name` (this is the same as `charge.offence_name`) and `charge.offence.offence_class`